dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgPanel.vue

```diff
@@ -113,6 +113,7 @@ export default defineComponent({
         kpiKey: kpiKey,
         kpiLabel: kpiField?.label ? kpiField.label : kpiKey,
         kpiDescription: kpiField?.description ? kpiField.description : "",
+        kpiFormFieldComponent: kpiField?.component ?? "",
         [dataIdOfLksgDataset]: kpiValue,
       } as KpiDataObject;
       let existingKpi = this.kpiDataObjects.get(kpiKey);
```

Comment:
The method `sortReportingPeriodsToDisplayAsColumns()` has a strange name for its input param.  It says `listOfDataDateToDisplayAsColumns`, even though the list contains column identifier objects.

Reply:
refactored to the type `DataSetReportingPeriod` , see above.

Reply:
I replied to the refactoring with a small change request in the conversation above.

Will keep this conversation here also open until the one above is closed.

Reply:
Problem was fixed in the conversation above, therefore closing this one here too